src/cashier/withdraw.es6

```diff
@@ -7,12 +7,11 @@ import liveapi from 'websockets/binary_websockets';
 import windows from 'windows/windows';
 import rv from 'common/rivetsExtra';
 import currencyDialog from 'cashier/currency';
-import _ from 'lodash';
+import {debounce} from 'lodash'
```

Comment:
@4p00rv doing `import _ from 'lodash'` and `import {debounce} from 'lodash'` would result in same size file. If you do `import debounce from 'lodash/debounce'`, that would save lot of file size. I have used that in many projects. You can also check the node_modules/lodash folder and find out more about it.